pkg/util/log/log.go

```diff
@@ -63,6 +89,24 @@ func CheckFatal(location string, err error) {
 		}
 		// %+v gets the stack trace from errors using github.com/pkg/errors
 		logger.Log("err", fmt.Sprintf("%+v", err))
+
+		if err = Flush(); err != nil {
+			fmt.Fprintln(os.Stderr, "Could not flush logger", err)
+		}
 		os.Exit(1)
 	}
 }
+
+type Flusher interface {
+	Flush() error
+}
```

Comment:
I think this is unused, is it?

Reply:
It is, see

```
func (l *LineBufferedLogger) Flush() error {
```

Reply:
But we don't use the interface anywhere, right? The variable is defined as a specific type.

Reply:
I also noticed it's unused, can we drop it?

Reply:
Yup, removed